package/utils/helpers.js

```
@@ -41,18 +41,21 @@
 }
 
-const packageMajorVersion = (packageName) => {
+const packageFullVersion = (packageName) => {
   // eslint-disable-next-line import/no-dynamic-require
   const packageJsonPath = require.resolve(`${packageName}/package.json`)
   // eslint-disable-next-line import/no-dynamic-require, global-require
-  return require(packageJsonPath).version.match(/^\d+/)[0]
+  return require(packageJsonPath).version
 }
 
+const packageMajorVersion = (packageName) => packageFullVersion(packageName).match(/^\d+/)[0]
```

ESLint

[error] Insert `⏎·`

(prettier/prettier)